### PR TITLE
Fix issue #134 with tempfile on windows

### LINKED ISSUE #134: Win Bin - Firefox not working anymore?
seeing this constantly whilst running --dpapi

image

### PR DESCRIPTION
**Pull request by NeffIsBack**
Fix #134:
Tempfile adds a handy attribute to the file which deletes the file after closing. This prevents sqlite3 on windows to connect to the file unfortunetly. This is to be expected according to the documentation https://docs.python.org/3/library/tempfile.html#tempfile.NamedTemporaryFile

In python 3.12 there is apperently a better solution, but for now removing the file manually is probably the best solution

### CODE REVIEW

**Comment by mpgn** [comment]
This is specific to windows ?

**Comment by NeffIsBack** [comment]
> This is specific to windows ?

Yes, reproducible on windows but working flawlessly on linux

File: nxc/protocols/smb/firefox.py [hunk 1]
```diff
@@ -4,6 +4,7 @@
 import hmac
 import json
 import ntpath
+import os
```
Comment (mpgn) [inline nxc/protocols/smb/firefox.py:7]: `from os import remove` to no ?

Comment (NeffIsBack) [reply nxc/protocols/smb/firefox.py:7]: Yep, thats better 👍

**Review by mpgn — APPROVED** [review]

**Comment by Marshall-Hallenbeck** [comment]
Ran tests and they passed 👍

**Pull request merged**